admin_manual/release_notes.rst

```diff
@@ -16,6 +17,102 @@ Release Notes
+
+LDAP-related improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~
+- When disabling or deleting user accounts in LDAP the administrator can choose to either *delete* or *disable* respective accounts in ownCloud when executing ``occ user:sync`` (``-m, --missing-account-action=MISSING-ACCOUNT-ACTION``). User accounts that are disabled in ownCloud can now be re-enabled automatically when running ``occ user:sync`` in case they are enabled in LDAP. When this behavior is desired administrators just need to add the ``-r, --re-enable`` option to their cron jobs or when manually executing ``occ user:sync``.
+- Furthermore it is now possible to execute ``occ user:sync`` only for *single* (``-u, --uid=UID``) or *seen* (``-s, --seenOnly``) users (users that are present in the database and have logged in at least once). These new options provide more granularity for administrators in terms of managing ``occ user:sync`` performance. Another notable change in behavior of ``occ user:sync`` is that administrators now have to explicitly specify the option ``-c, --showCount`` to display the number of users to be synchronized.
+
+New events for audit logging
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+New events have been added to be used for audit logging, among others. These include *configuration changes* by administrators and users, *file comments* (*add/edit/delete*) and *updating existing public links*. When logs are forwarded to external analyzers like Splunk, administrators can check to add the new events. The latest version of the Auditing extension (*admin_audit*) is required.
+
+New command to verify and repair file checksums
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+With ownCloud 10 file integrity checking by computing and matching checksums has been introduced to ensure that transferred files arrive at their target in the exact state as their origin. In some rare cases wrong checksums can be written to the database leading to synchronization issues with e.g. the Desktop Client. To mitigate such situations a new command ``occ files:checksums:verify`` has been introduced. Executing the command will recalculate checksums for all files of a user or within a specified path on the storage and compare them with the values in the database. Naturally the command also offers an option to repair incorrect checksum values (``-r, --repair``). Please check the available options by executing ``occ files:checksums:verify --help``. Note: Executing this command might take some time depending on the file count.
```

Comment:
integrity check exists since OC 9.0

Reply:
"Naturally" here sounds German/French. Is that correct English as well @phil-davis ?

Reply:
"Naturally" could be removed. Just start "The command also offers..."

Reply:
There's nothing, specifically, wrong with using it, @PVince81.  It feels like it assumes that it is normal to expect that functionality, however.

Reply:
s/Executing the command will recalculate checksums for all files of a user or within a specified path on the storage and compare them with the values in the database/The command recalculates checksums either for all files of a user or for files within a specified path, and compares them with the values in the database./g

Reply:
done